app/scripts/directives/unbindService.js

```diff
@@ -0,0 +1,121 @@
+'use strict';
+
+(function() {
+
+  angular.module('openshiftConsole').component('unbindService', {
+    controller: [
+      '$filter',
+      'BindingModalUtils',
+      'DataService',
+      'ServiceInstancesService',
+      UnbindService
+    ],
+    controllerAs: 'ctrl',
+    bindings: {
+      target: '<',
+      bindings: '<',
+      applications: '<',
```

Comment:
`bindings` and `applications` are maps. Looking at just the names, I thought these were flat lists. Can we use similar names to what we use on the overview so it's clear they are the same maps?